src/integrationTest/java/org/opensearch/test/framework/cluster/LocalCluster.java

```diff
@@ -141,7 +141,6 @@ public void before() {
             }
 
             for (Map.Entry<String, LocalCluster> entry : remotes.entrySet()) {
-                @SuppressWarnings("resource")
```

Comment:
Yes, since we check it in our gradle:
https://github.com/opensearch-project/security/blob/6d35a0af9de48620f9c6d5b2c67b094713ca8c3e/build.gradle#L100

otherwise build fails.